lan867x/CMakeLists.txt

```
@@ -0,0 +1,3 @@
+idf_component_register(SRCS "src/esp_eth_phy_lan867x.c"
+                       INCLUDE_DIRS "include"
+                       PRIV_REQUIRES "log esp_eth")
```

isn't this treated like one component with a space? suggest removing the quotes.
```suggestion
                       PRIV_REQUIRES log esp_eth)
```